src/app/find/mapPage/filters/filters.css

```diff
@@ -0,0 +1,69 @@
+@value primaryColor from "../../../../colors.css";
+
+.doneBtnContainer{
+	position: fixed;
+	bottom: 0;
+	width: 100%;
+	z-index: 1040;
+}
+
+.doneBtn{
+	border: 2px solid primaryColor;
+	background-color: #ffffff;
+	min-height: 56px;
+	width: 96%;
```

Comment:
96%? Sounds like maybe this should just be a fluid button, with some smalll margins on its container, no?

Reply:
I'm not following.

Reply:
I mean to say our Button component takes a property called "fluid", which makes it 100% width. So, if you take e.g. a `<Button />`, and make it `<Button fluid />`, you'll get pretty much the desired effect.
Except, it seems you want some slight spacing around it, which is why it's 96% and not 100%. In that case, actually putting padding or margins or whatnot on whatever element contains this button would be the most straightforward way to convey that intention.

In general, this kind of division between component styling (e.g. the size of the button, which is part of Button.css) and layout styling (e.g. what happens around it and where does it reside, which is part of filters.css) makes it easier to maintain reusable components with a standard appearance, and also to get a clear image of what a page should look like by going over its code.

(Having said that, this of course isn't a very major comment, and makes no functional difference.)